firefox-ios/firefox-ios-tests/Tests/XCUITests/LoginsTests.swift

```diff
@@ -121,16 +121,7 @@ class LoginTest: BaseTestCase {
             openLoginsSettings()
             mozWaitForElementToExist(app.tables["Login List"])
             mozWaitForElementToExist(app.staticTexts[domain])
-            // XCTAssertTrue(app.staticTexts[domainSecondLogin].exists)
-            // Workaround for Bitrise specific issue. "vagrant" user is used in Bitrise.
-            if (ProcessInfo.processInfo.environment["HOME"]!).contains(String("vagrant")) {
-                XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
-            // Workaround for Github Actions specific issue. "runner" user is used in Github Actions.
-            } else if (ProcessInfo.processInfo.environment["HOME"]!).contains(String("runner")) {
-                XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
-            } else {
-                XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
-            }
+            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
```

Comment:
@isabelrios The conditional check was needed before because in Bitrise keychain is not accessible and the code failed silently on writes. So everytime the test attempted to add a login it generated a new encryption key and deleted the old logins, resulting in having only one login stored at a time on BR. Now that I am mocking keychain the key should be retrieved so we have access to both logins ( or any n number of logins ) on CI. 
( confirmed by adding some debug logs on another BR run ⏬ )
![Screenshot 2025-01-23 at 01 20 20](https://github.com/user-attachments/assets/1afc396d-9746-4b98-89eb-3dc5e635d191)

Reply:
great, thank you!